diff --color -urN wxWidgets-323a465e577e03f330e2e6a4c78e564d125340cb/build/cmake/lib/webview/CMakeLists.txt dep_wxWidgets/build/cmake/lib/webview/CMakeLists.txt
--- wxWidgets-323a465e577e03f330e2e6a4c78e564d125340cb/build/cmake/lib/webview/CMakeLists.txt	2024-06-26 10:39:50
+++ dep_wxWidgets/build/cmake/lib/webview/CMakeLists.txt	2025-01-03 22:01:59
@@ -46,9 +46,9 @@
 elseif(WXMSW)
     if(wxUSE_WEBVIEW_EDGE)
         # Update the following variables if updating WebView2 SDK
-        set(WEBVIEW2_VERSION "1.0.705.50")
+        set(WEBVIEW2_VERSION "1.0.2592.51")
         set(WEBVIEW2_URL "https://www.nuget.org/api/v2/package/Microsoft.Web.WebView2/${WEBVIEW2_VERSION}")
-        set(WEBVIEW2_SHA256 "6a34bb553e18cfac7297b4031f3eac2558e439f8d16a45945c22945ac404105d")
+        set(WEBVIEW2_SHA256 "805c79e05184fab18c9fe7b8ba820c598399b97adc1fbf5b0ea490efad91d5b8")
 
         set(WEBVIEW2_DEFAULT_PACKAGE_DIR "${CMAKE_BINARY_DIR}/packages/Microsoft.Web.WebView2.${WEBVIEW2_VERSION}")
 
diff --color -urN wxWidgets-323a465e577e03f330e2e6a4c78e564d125340cb/include/wx/generic/private/widthcalc.h dep_wxWidgets/include/wx/generic/private/widthcalc.h
--- wxWidgets-323a465e577e03f330e2e6a4c78e564d125340cb/include/wx/generic/private/widthcalc.h	2024-06-26 10:39:50
+++ dep_wxWidgets/include/wx/generic/private/widthcalc.h	2025-01-03 22:01:59
@@ -101,9 +101,9 @@
                 UpdateWithRow(row);
             }
 
-            wxLogTrace("items container",
-                       "determined best size from %zu top, %zu bottom "
-                       "plus %zu more visible items out of %zu total",
+            wxLogTrace(wxString::FromAscii("items container"),
+                       wxString::FromAscii("determined best size from %zu top, %zu bottom "
+                       "plus %zu more visible items out of %zu total"),
                        top_part_end,
                        count - bottom_part_start,
                        last_visible - first_visible,
diff --color -urN wxWidgets-323a465e577e03f330e2e6a4c78e564d125340cb/include/wx/variant.h dep_wxWidgets/include/wx/variant.h
--- wxWidgets-323a465e577e03f330e2e6a4c78e564d125340cb/include/wx/variant.h	2024-06-26 10:39:50
+++ dep_wxWidgets/include/wx/variant.h	2025-01-03 22:01:59
@@ -517,7 +517,7 @@
 \
 expdecl classname& operator << ( classname &value, const wxVariant &variant )\
 {\
-    wxASSERT( variant.GetType() == #classname );\
+    wxASSERT( variant.GetType() == wxString::FromAscii(#classname) );\
     \
     classname##VariantData *data = (classname##VariantData*) variant.GetData();\
     value = data->GetValue();\
